Importing a config_map volume with no mode causes panic

When importing a resource with a config_map volume that has items with no mode set, the provider panics.

### Terraform Version
Terraform v0.11.10

### Affected Resource(s)
Anything that uses a pod with config map items for volume mounts

### Terraform Configuration Files
```hcl
resource "kubernetes_stateful_set" "myapp" {
  metadata {
    name      = "myapp"
    namespace = "default"
  }

  spec {
    replicas     = 3

    update_strategy {
      type = "RollingUpdate"
    }

    selector {
      app = "myapp"
    }

    template {
      metadata {
        labels {
          app = "myapp"
        }
      }

      spec {
        container {
          name  = "myapp"
          image = "myapp:latest"

          port {
            name           = "http"
            container_port = 80
          }

          volume_mount {
            name       = "config"
            mount_path = "/etc/myapp"
          }

        }

        volume {
          name = "config"

          config_map {
            name = "${kubernetes_config_map.myapp.metadata.0.name}"

            items {
              key  = "file1"
              path = "file1.conf"
            }

            items {
              key  = "file2"
              path = "file2.conf"
            }
          }
        }
      }
    }
  }
}
```

### Debug Output
See Panic Output

### Panic Output
```
Error: kubernetes_stateful_set.myapp (import id: default/myapp): 1 error(s) occurred:

* import kubernetes_stateful_set.myapp result: deafult/myapp: kubernetes_stateful_set.myapp: unexpected EOF


panic: runtime error: invalid memory address or nil pointer dereference
2018-11-12T10:53:26.473Z [DEBUG] plugin.terraform-provider-kubernetes: [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1334012]
2018-11-12T10:53:26.473Z [DEBUG] plugin.terraform-provider-kubernetes: 
2018-11-12T10:53:26.473Z [DEBUG] plugin.terraform-provider-kubernetes: goroutine 87 [running]:
2018-11-12T10:53:26.473Z [DEBUG] plugin.terraform-provider-kubernetes: github.com/sl1pm4t/terraform-provider-kubernetes/kubernetes.flattenConfigMapVolumeSource(0xc420933840, 0xc4209e3f80, 0x16883fb, 0x4)
2018-11-12T10:53:26.474Z [DEBUG] plugin.terraform-provider-kubernetes: 	REMOVED/src/go/src/github.com/sl1pm4t/terraform-provider-kubernetes/kubernetes/structures_pod.go:351 +0x2f2
2018-11-12T10:53:26.474Z [DEBUG] plugin.terraform-provider-kubernetes: github.com/sl1pm4t/terraform-provider-kubernetes/kubernetes.flattenVolumes(0xc4204d6800, 0x2, 0x4, 0x20, 0xc4203e21b8, 0x0, 0x0, 0x0)
2018-11-12T10:53:26.474Z [DEBUG] plugin.terraform-provider-kubernetes: 	REMOVED/src/go/src/github.com/sl1pm4t/terraform-provider-kubernetes/kubernetes/structures_pod.go:208 +0x3674
2018-11-12T10:53:26.474Z [DEBUG] plugin.terraform-provider-kubernetes: github.com/sl1pm4t/terraform-provider-kubernetes/kubernetes.flattenPodSpec(0xc4204d6800, 0x2, 0x4, 0x0, 0x0, 0x0, 0xc420398500, 0x1, 0x4, 0xc4204e95b0, ...)
2018-11-12T10:53:26.474Z [DEBUG] plugin.terraform-provider-kubernetes: 	REMOVED/src/go/src/github.com/sl1pm4t/terraform-provider-kubernetes/kubernetes/structures_pod.go:88 +0x705
2018-11-12T10:53:26.474Z [DEBUG] plugin.terraform-provider-kubernetes: github.com/sl1pm4t/terraform-provider-kubernetes/kubernetes.flattenStatefulSetSpec(0xc4204e92f8, 0xc420a459c0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
2018-11-12T10:53:26.474Z [DEBUG] plugin.terraform-provider-kubernetes: 	REMOVED/src/go/src/github.com/sl1pm4t/terraform-provider-kubernetes/kubernetes/structures_stateful_set.go:27 +0x406
2018-11-12T10:53:26.474Z [DEBUG] plugin.terraform-provider-kubernetes: github.com/sl1pm4t/terraform-provider-kubernetes/kubernetes.resourceKubernetesStatefulSetRead(0xc420a7ad20, 0x15397e0, 0xc420833ef0, 0xc420a7ad20, 0x0)
2018-11-12T10:53:26.474Z [DEBUG] plugin.terraform-provider-kubernetes: 	REMOVED/src/go/src/github.com/sl1pm4t/terraform-provider-kubernetes/kubernetes/resource_kubernetes_stateful_set.go:250 +0x589
2018-11-12T10:53:26.475Z [DEBUG] plugin.terraform-provider-kubernetes: github.com/sl1pm4t/terraform-provider-kubernetes/vendor/github.com/hashicorp/terraform/helper/schema.(*Resource).Refresh(0xc42077d500, 0xc4202333b0, 0x15397e0, 0xc420833ef0, 0xc42078c5e0, 0x4bf301, 0x13e2fc0)
2018-11-12T10:53:26.479Z [DEBUG] plugin.terraform-provider-kubernetes: 	REMOVED/src/go/src/github.com/sl1pm4t/terraform-provider-kubernetes/vendor/github.com/hashicorp/terraform/helper/schema/resource.go:354 +0x167
2018-11-12T10:53:26.479Z [DEBUG] plugin.terraform-provider-kubernetes: github.com/sl1pm4t/terraform-provider-kubernetes/vendor/github.com/hashicorp/terraform/helper/schema.(*Provider).Refresh(0xc42077d650, 0xc420233360, 0xc4202333b0, 0x18, 0x18, 0xc42090ff80)
2018-11-12T10:53:26.479Z [DEBUG] plugin.terraform-provider-kubernetes: 	REMOVED/src/go/src/github.com/sl1pm4t/terraform-provider-kubernetes/vendor/github.com/hashicorp/terraform/helper/schema/provider.go:308 +0x9a
2018-11-12T10:53:26.480Z [DEBUG] plugin.terraform-provider-kubernetes: github.com/sl1pm4t/terraform-provider-kubernetes/vendor/github.com/hashicorp/terraform/plugin.(*ResourceProviderServer).Refresh(0xc4201ca940, 0xc42034a990, 0xc42034abb0, 0x0, 0x0)
2018-11-12T10:53:26.481Z [DEBUG] plugin.terraform-provider-kubernetes: 	REMOVED/src/go/src/github.com/sl1pm4t/terraform-provider-kubernetes/vendor/github.com/hashicorp/terraform/plugin/resource_provider.go:549 +0x4e
2018-11-12T10:53:26.481Z [DEBUG] plugin.terraform-provider-kubernetes: reflect.Value.call(0xc420181200, 0xc4201cc4e0, 0x13, 0x1687e4f, 0x4, 0xc42007ff18, 0x3, 0x3, 0xc4204ea180, 0x0, ...)
2018-11-12T10:53:26.481Z [DEBUG] plugin.terraform-provider-kubernetes: 	REMOVED/bin/go/src/reflect/value.go:447 +0x969
```

### Expected Behavior
Should have imported the stateful set successfully with no mode set on the items

### Actual Behavior
panic from the provider as it references a pointer to a missing variable

### Steps to Reproduce
1. Create a resource outside of terraform that has the config_map. Make sure not to set a mode in the items! It must be created outside of terraform as the provider itself seems to set the mode to 0 by default.
2. Try to import it `terraform import kubernetes_stateful_set.myapp default/myapp`

### Important Factoids
Only happens when importing a resource created outside of terraform. The provider sets the mode to 0 if you don't set it so the issue doesn't happen.

--- a/kubernetes/structures_pod.go
+++ b/kubernetes/structures_pod.go
@@ -345,7 +345,9 @@ func flattenConfigMapVolumeSource(in *v1.ConfigMapVolumeSource) []interface{} {
 		for i, v := range in.Items {
 			m := map[string]interface{}{}
 			m["key"] = v.Key
-			m["mode"] = int(*v.Mode)
+			if v.Mode != nil {
+				m["mode"] = int(*v.Mode)
+			}
 			m["path"] = v.Path
 			items[i] = m
 		}

